<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QDialog" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>538</width>
    <height>388</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Verdana</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>Взаимодействие с массивами</string>
  </property>
  <widget class="QPushButton" name="RandomValuesButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>170</y>
     <width>181</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Случайные числа</string>
   </property>
  </widget>
  <widget class="QLabel" name="GreetingLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>171</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Добро пожаловать, Гость!</string>
   </property>
  </widget>
  <widget class="QPushButton" name="SaveSortedArrayButton">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>350</y>
     <width>121</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Сохранить</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="NumberOfArrayElementsLineEdit">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>80</y>
     <width>181</width>
     <height>28</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="PlusButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>120</y>
     <width>81</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>+</string>
   </property>
  </widget>
  <widget class="QPushButton" name="SaveArrayButton">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>350</y>
     <width>121</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Сохранить</string>
   </property>
  </widget>
  <widget class="QPushButton" name="SortButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>212</y>
     <width>181</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Сортировка</string>
   </property>
  </widget>
  <widget class="QListView" name="ArrayListview">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>20</y>
     <width>121</width>
     <height>321</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="LoadArraysButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>181</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Сохраненные массивы</string>
   </property>
  </widget>
  <widget class="QPushButton" name="MinusButton">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>120</y>
     <width>81</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>-</string>
   </property>
  </widget>
  <widget class="QListView" name="SortedArrayListview">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>20</y>
     <width>121</width>
     <height>321</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="QuitButton">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>340</y>
     <width>31</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: none;
background: transparent;
padding: 0px;
margin: 0px;
outline: none;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>photos/door_exit.png</normaloff>photos/door_exit.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>40</width>
     <height>40</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="DarkThemeButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>340</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: none;
background: transparent;
padding: 0px;
margin: 0px;
outline: none;
</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>photos/darkth.png</normaloff>photos/darkth.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>40</width>
     <height>40</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="InfoButton">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>340</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: none;
background: transparent;
padding: 0px;
margin: 0px;
outline: none;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>photos/angle.png</normaloff>photos/angle.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>40</width>
     <height>40</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="LeaveButton">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>340</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: none;
background: transparent;
padding: 0px;
margin: 0px;
outline: none;
</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>photos/leftacc.png</normaloff>photos/leftacc.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>40</width>
     <height>40</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="FocusButton">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>0</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: none;
background: transparent;
padding: 0px;
margin: 0px;
outline: none;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="iconSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
